docker-compose.yml

```diff
@@ -34,35 +36,40 @@ services:
       - ./docker/app/storage:/var/www/app/storage:rw,delegated
     depends_on:
       - db
-    networks: 
-      - invoiceninja  
+    networks:
+      - default
     extra_hosts:
-      - "in5.localhost:192.168.0.124 " #host and ip
+      - "${CNAME}:192.168.0.124 " #host and ip
 
   db:
+    container_name: invoice_ninja5_db
     image: mysql:5
-    ports:
-      - "3305:3306"
+    env_file: .env
     restart: always
-    environment: 
-      - MYSQL_ROOT_PASSWORD=ninjaAdm1nPassword
-      - MYSQL_USER=ninja
-      - MYSQL_PASSWORD=ninja
-      - MYSQL_DATABASE=ninja
+    environment:
+      - MYSQL_ROOT_PASSWORD=${DB_PASSWORD1} # config for 5
+      - MYSQL_USER=${DB_USERNAME1}
+      - MYSQL_PASSWORD=${DB_PASSWORD1}
+      - MYSQL_DATABASE=${DB_DATABASE1}
+      # - MYSQL_ROOT_PASSWORD=${DB_PASSWORD} # config for 4
+      # - MYSQL_USER=${DB_USERNAME}
+      # - MYSQL_PASSWORD=${DB_PASSWORD}
+      # - MYSQL_DATABASE=${DB_DATABASE}
     volumes:
       - ./docker/mysql/data:/var/lib/mysql:rw,delegated
     networks:
-      - invoiceninja
+      - default
     extra_hosts:
-      - "in5.localhost:192.168.0.124 " #host and ip
+      - "${CNAME}:192.168.0.124 " #host and ip
+
 
   # THIS IS ONLY A VALID CONFIGURATION FOR IN 4. DO NOT USE FOR IN 5.
   # cron:
   #   image: invoiceninja/invoiceninja:alpine-4
   #   volumes:
-      # - ./docker/app/public:/var/www/app/public:rw,delegated
-      # - ./docker/app/storage:/var/www/app/storage:rw,delegated
-      # - ./docker/app/public/logo:/var/www/app/public/logo:rw,delegated
+  # - ./docker/app/public:/var/www/app/public:rw,delegated
+  # - ./docker/app/storage:/var/www/app/storage:rw,delegated
+  # - ./docker/app/public/logo:/var/www/app/public/logo:rw,delegated
```

Comment:
Could you add the correct spaces between the "#" and "-", so when you uncomment the whole block the YAML is correctly indented.